apps/referral/constants/defaults.py

```diff
@@ -0,0 +1,16 @@
+# Referral app settings
+
+# Allow hash referral codes
+HTK_REFERRAL_ALLOW_HASH_CODE = True
+
+# Allow usernames as referral code
+HTK_REFERRAL_ALLOW_USERNAME = True
+
+# Redirect URL name for the referral URL
+HTK_REFERRAL_REDIRECT_URL_NAME = 'home'
```

Comment:
Set it to None and require the developer to configure something. If it's not set, redirect to `/`